public/styles.css

```diff
@@ -133,7 +135,9 @@ body {
 }
 
 .overview-widget-productInfo {
+  font-size: large;
   min-height: 150px;
+  width: 340px;
```

Comment:
it is not good to use px as they are not responsive to screen-sizes. all pixels should be declared as 'rem' in relation to the global font-size.